### ISSUE TITLE
Please fix the bug that is causing encode errors in keypoints

### ISSUE THREAD

**Issue by doudoudiule** [zju3dv/EasyMocap#354]
# TODO: bug: This encode will arise errors in keypoints
  kpts1 = body_model.keypoints(body_params, return_tensor=False)[0]

**Comment by chingswy** [comment]
I can't reproduce this bug.

**Comment by doudoudiule** [comment]
Thank you for your response, but I noticed in your code that this bug is indicated . and The bug is right in the code file 'EasyMocap\easymocap\estimator\SPIN\spin_api.py' at line 304. # TODO: bug: This encode will arise errors in keypoints

**Comment by chingswy** [comment]
You can follow the newest code [here](https://chingswy.github.io/easymocap-public-doc/develop/03_fitsmpl_monocular.html)

**Comment by doudoudiule** [comment]
I have a different perspective on this. I believe there might be room for improvement in terms of modesty. While reviewing your project, I did come across a bug. However, I understand that you might have a differing viewpoint on whether it's a bug or not. I used to hold a high opinion of this project, and while my stance has evolved, I've decided to explore the use of MMHuman3d due to the enhanced solutions they offer.
